banzai/qc/pattern_noise.py

```diff
@@ -39,7 +38,9 @@ def do_stage(self, images):
                 images_to_remove.append(image)
             else:
                 self.logger.info('No pattern noise found.', extra=logging_tags)
-            self.save_qc_results({'PatternNoise': pattern_noise_is_bad}, image)
+            self.save_qc_results({'pattern_noise.failed': pattern_noise_is_bad,
+                                  'pattern_noise.snr_threshold': self.SNR_THRESHOLD,
+                                  'pattern_noise.pixel_threshold': self.PIXEL_THRESHOLD}, image)
```

Comment:
This isn't really a number of pixels in the normal sense. This is actually the number of bins in a stacked 2d FFT. Maybe we should call this freq_bins_threshold?

Reply:
I agree, I've changed it to your suggested name